packages/plugin-eslint-rules/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@grafana/plugin-eslint-rules",
+  "version": "0.0.4",
```

Comment:
With the rename of the package we can reset this to `0.0.0`.

Reply:
Fixed 👍